apps/prairielearn/src/pages/instructorStudents/instructorStudents.html.tsx

```diff
@@ -82,24 +81,25 @@ function StudentsCard({ course, courseInstance, students, timezone }: StudentsCa
   const [columnFilters, setColumnFilters] = useState<ColumnFiltersState>([]);
   const [columnSizing, setColumnSizing] = useState<ColumnSizingState>({});
 
-  const columns = useMemo<AccessorKeyColumnDef<StudentRow, any>[]>(
+  const columns = useMemo(
     () => [
-      columnHelper.accessor('user.uid', {
+      columnHelper.accessor((row) => row.user.uid, {
```

Comment:
cc @reteps this is the more type-safe variant of an accessor. with this, `info.getValue()` reflects the actual type of the data.

Dropping the explicit type on the `useMemo` and relying just on type inference was also necessary to get this working.